src/shared/components/FragmentShareModal/EmbedContent.tsx

```diff
@@ -0,0 +1,363 @@
+					</ExpandableContainer>
+				</div>
+			);
+		}
+
+		if (descriptionType === EmbedCodeDescriptionType.CUSTOM) {
+			const controls: RichTextEditorControl[] = [
+				'undo',
+				'redo',
+				'separator',
+				'bold',
+				'italic',
+				'strike-through',
+				'underline',
+				'separator',
+				'remove-styles',
+				'separator',
+				'text-align',
+				'separator',
+				'headings',
+				'list-ol',
+				'list-ul',
+				'separator',
+				'link',
+				'separator',
+				'fullscreen',
+			];
+			return (
+				<RichTextEditorWithInternalState
+					controls={controls}
+					enabledHeadings={['h3', 'h4', 'normal']}
+					value={description || ''}
+					disabled={!!generatedCode}
+					onChange={setDescription}
+				/>
+			);
+		}
+
+		return <></>;
+	};
+
+	const renderDescriptionWrapper = () => {
+		if (item.externalWebsite === EmbedCodeExternalWebsite.BOOKWIDGETS) {
+			return (
+				<Alert type="info">
+					<span className="c-content">
+						{tHtml(
+							'Bij het insluiten op Bookwidgets wordt er geen beschrijving bij het fragment weergegeven.'
+						)}
+					</span>
+				</Alert>
+			);
+		}
+
+		return (
+			<>
+				<ButtonGroup className="u-d-flex">
+					<Button
+						id="share-fragment-use-original-description"
+						type="secondary"
+						className="u-flex-auto"
+						disabled={!!generatedCode}
+						label={tText('Originele beschrijving')}
+						active={descriptionType === EmbedCodeDescriptionType.ORIGINAL}
+						onClick={() => handleDescriptionToggle(EmbedCodeDescriptionType.ORIGINAL)}
+					/>
+					<Button
+						type="secondary"
+						className="u-flex-auto"
+						disabled={!!generatedCode}
+						label={tText('Eigen beschrijving')}
+						active={descriptionType === EmbedCodeDescriptionType.CUSTOM}
+						onClick={() => handleDescriptionToggle(EmbedCodeDescriptionType.CUSTOM)}
+					/>
+					<Button
+						type="secondary"
+						className="u-flex-auto"
+						disabled={!!generatedCode}
+						label={tText('Geen beschrijving')}
+						active={descriptionType === EmbedCodeDescriptionType.NONE}
+						onClick={() => handleDescriptionToggle(EmbedCodeDescriptionType.NONE)}
+					/>
+				</ButtonGroup>
+				<Spacer margin="top">{renderDescription()}</Spacer>
+			</>
+		);
+	};
+
+	const renderRightSideFooter = () => {
+		if (onSave) {
+			return (
+				<ToolbarRight>
+					<ToolbarItem>
+						<ButtonToolbar>
+							<Button
+								type="primary"
+								label={tText('Opslaan')}
+								title={tText('Opslaan')}
+								ariaLabel={tText('Opslaan')}
+								onClick={handleSave}
+							/>
+						</ButtonToolbar>
+					</ToolbarItem>
+				</ToolbarRight>
+			);
+		}
+
+		if (generatedCode) {
+			return (
+				<ToolbarRight>
+					<ToolbarItem>{generatedCode}</ToolbarItem>
+					<ToolbarItem>
+						<ButtonToolbar>
+							<Button
+								type="primary"
+								icon={IconName.copy}
+								title={tText('Code kopiëren')}
+								ariaLabel={tText('Code kopiëren')}
```

Comment:
Om iets te kopieren wordt in avo meestal dupliceren gebruikt ipv kopieren
```suggestion
								title={tText('Code dupliceren')}
								ariaLabel={tText('Code dupliceren')}
```

Reply:
In dit geval is het echter wel kopiëren want het gaat erom dat je de code naar je clipboard kopieert:
https://meemoo.atlassian.net/wiki/spaces/AVO2/pages/4902650030/FA+Fragment+aanmaken+kopieren+en+bewerken+modal+-+BookWidgets